### DISCUSSION TITLE
如何使用maa运行肉鸽改过的json文件以及其他

### DISCUSSION THREAD

**Discussion by 820512356** [MaaAssistantArknights/MaaAssistantArknights#9125] [Q&A] [answered]
自己改写了一下萨米肉鸽维什戴尔开的战斗逻辑，尤其是第一层，但是写完后发现直接修改会导致maa报错无法使用，用小工具打开会显示无法识别。
在这里请问一下如何实验自己写的是否正常。

还有一点，如何让maa判断那关该带那个技能（肉鸽），还是说现在无法这样判断。以及如何让maa在什么时候开技能，我找了几个文件没见到类似的描述，在辅助协议中的“do”指令完全没找到，难道是默认有了就开？

**Comment by ABA2396** [comment, marked as answer]
打不开大概率是格式错了，因为多了少了字段都没啥影响，会有默认值。检查下是不是标点啥的写错了，比如多了少了括号，用了中文标点什么的，写json最好装个有json提示的文本编辑器

肉鸽的技能都是好了就开

**Reply by 820512356** [reply]
那是在小工具打开还是直接文件夹改然后正常运行呢？

**Reply by ABA2396** [reply]
> 那是在小工具打开还是直接文件夹改然后正常运行呢？
> 

直接运行，但最好自己做过备份，如果更新的时候对应的文件改了是会覆盖的
